kafka-connect-http/src/main/scala/io/lenses/streamreactor/connect/http/sink/HttpSinkTask.scala

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2017-2024 Lenses.io Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lenses.streamreactor.connect.http.sink
+
+import cats.effect.unsafe.implicits.global
+import cats.effect.IO
+import cats.effect.Ref
+import com.typesafe.scalalogging.LazyLogging
+import io.lenses.streamreactor.common.utils.AsciiArtPrinter.printAsciiHeader
+import io.lenses.streamreactor.common.utils.JarManifest
+import io.lenses.streamreactor.connect.cloud.common.model.Offset
+import io.lenses.streamreactor.connect.cloud.common.model.Topic
+import io.lenses.streamreactor.connect.cloud.common.model.TopicPartition
+import io.lenses.streamreactor.connect.http.sink.config.HttpSinkConfig
+import io.lenses.streamreactor.connect.http.sink.config.HttpSinkConfig.fromJson
+import io.lenses.streamreactor.connect.http.sink.config.HttpSinkConfigDef.configProp
+import io.lenses.streamreactor.connect.http.sink.tpl.RawTemplate
+import io.lenses.streamreactor.connect.http.sink.tpl.TemplateType
+import org.apache.kafka.clients.consumer.OffsetAndMetadata
+import org.apache.kafka.common.{ TopicPartition => KafkaTopicPartition }
+import org.apache.kafka.connect.errors.ConnectException
+import org.apache.kafka.connect.sink.SinkRecord
+import org.apache.kafka.connect.sink.SinkTask
+
+import java.util
+import scala.jdk.CollectionConverters.IterableHasAsScala
+import scala.jdk.CollectionConverters.MapHasAsJava
+import scala.jdk.CollectionConverters.MapHasAsScala
+
+class HttpSinkTask extends SinkTask with LazyLogging {
+  private val manifest = JarManifest(getClass.getProtectionDomain.getCodeSource.getLocation)
+
+  implicit val ec: scala.concurrent.ExecutionContext = scala.concurrent.ExecutionContext.global
```

Comment:
this is unnused. 

Also, to make things more obvious using `implicit val runtime = IORuntime.global` rather than using the import is a bit better

Reply:
I think if you are missing the implicit, this is what IntelliJ suggests you add.  Will have to remember not to go with the default suggestion!

Reply:
It's not a major problem.. It just highlights the fact that we're "using" the IORuntime here.. 

Imports are often overlooked and or not read so it makes it a bit harder to understand where the runtime instance is coming from(I may have been spending too much time tracking this things lately so that's why I'm a bit sensitive :wink: )